### DISCUSSION TITLE
[3주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#31] [General]
### 진행순서
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [Java] Object, Class, Instance #22 
 * [Algorithm] Merge Sort #29 
 * [Algorithm] Quick Sort #30 
2. 이주형
 * [Database] 트랜잭션 #24 
3. 김정윤
 * [Java/Algorithm] 최소비용 신장트리(MST) #26

**Reply by leeejuhyeong** [reply]
트랜잭션 참고자료 : https://m.blog.naver.com/good_ray/221943028058

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😂
* 좋았던 점
* 건의사항
* 느낀점

**Reply by ChaerinYu** [reply]
노곤노곤하네요 ㅠㅠ 오늘도 고생 많으셨습니다~!~

**Reply by KJY97** [reply]
- 좋았던 점
  퀵정렬이랑 합병정렬 많이 어렵네요.. 정리해주셔서 감사합니다! 
- 다익스트라 알고리즘과 프림 알고리즘도 정리해서 나중에 올릴게요!

**Reply by leeejuhyeong** [reply]
- 좋았던점 : 크루스칼이랑 프림.. 내용정리해야지, 퀵정렬, 병합정렬 정리해야지 했는데 두분이해주셧어요.. 가려운 곳을 긁어주신 기분입니다.
